requirements.txt

```diff
@@ -1,7 +1,8 @@
 # Runtime requirements for the python 3 version of cirq.
 
 dataclasses; python_version < '3.7'
-google-api-python-client~=1.6
+enum34; python_version < "3.4"
```

Comment:
I don't think we (or anyone) supports python < 3.4, so the enum backport is probably unnecessary

Reply:
Thanks. Removed.